marimo/_server/api/endpoints/ws.py

```diff
@@ -651,7 +652,16 @@ def get_session() -> Session:
             self.status = ConnectionState.OPEN
             # if auto-instantiate if false, replay the previous session
             if not self.auto_instantiate:
-                new_session.sync_session_view_from_cache()
+                from marimo import __version__
+
+                app = new_session.app_file_manager.app
+                codes = tuple(
+                    cell_data.code
+                    for cell_data in app.cell_manager.cell_data()
+                )
```

Comment:
can these lines be pushed into `sync_session_view_from_cache`?

```
                from marimo import __version__

                app = new_session.app_file_manager.app
                codes = tuple(
                    cell_data.code
                    for cell_data in app.cell_manager.cell_data()
                )
```

Reply:
Done, also updated the tests, realized they weren't testing what I wanted